PIL/PngImagePlugin.py

```diff
@@ -249,7 +249,7 @@ def add_itxt(self, key, value, lang="", tkey="", zip=False):
             self.add(b"iTXt", key + b"\0\0\0" + lang + b"\0" + tkey + b"\0" +
                      value)
 
-    def add_text(self, key, value, zip=0):
+    def add_text(self, key, value, zip=False):
```

Comment:
As an aside, this is a really badly named parameter, as it shadows a builtin. However, since it's (nominally) a named parameter in a public interface, changing it is potentially going to cause more pain than it's worth.

Reply:
I agree, but I wasn't attempting to change the public interface. I agree, that may be worthwhile to avoid the builtin.

Reply:
Yep.